src/redisearch_rs/inverted_index_bencher/src/bencher.rs

```diff
@@ -96,51 +45,313 @@ impl NumericBencher {
     }
 
     pub fn encoding(&self, c: &mut Criterion) {
-        for (group, values) in &self.encoding_test_values {
-            let group = format!("Encode - {}", group);
+        for BenchGroup { group, inputs } in &self.test_values {
+            let group = format!("Encode | {}", group);
```

Comment:
```suggestion
            let group = format!("Encode Numeric | {group}");
```

I think all the benches should be fully namespaced to prevent clash with the rest of the enc/dec benches?